pkg/apis/certificates/v1beta1/defaults_test.go

```diff
@@ -40,27 +40,19 @@ func TestIsKubeletServingCSR(t *testing.T) {
 		return csr
 	}
 	tests := map[string]struct {
-		req                               *x509.CertificateRequest
-		usages                            []capi.KeyUsage
-		allowOmittingUsageKeyEncipherment bool
-		exp                               bool
+		req    *x509.CertificateRequest
+		usages []capi.KeyUsage
+		exp    bool
 	}{
 		"defaults for kubelet-serving": {
 			req:    newCSR(kubeletServerPEMOptions),
 			usages: kubeletServerUsages,
 			exp:    true,
 		},
-		"defaults without key encipherment for kubelet-serving if allow omitting key encipherment": {
```

Comment:
I expected to keep these testcases and just drop the allowOmittingUsageKeyEncipherment field from the testcase

Reply:
```
"defaults for kubelet-serving if allow omitting key encipherment": {
			req:                               newCSR(kubeletServerPEMOptions),
			usages:                            kubeletServerUsages,
			allowOmittingUsageKeyEncipherment: true,
			exp:                               true,
		},
```
is dupped with
```
		"defaults for kubelet-serving": {
			req:    newCSR(kubeletServerPEMOptions),
			usages: kubeletServerUsages,
			exp:    true,
		},
```